rules/aws_cloudtrail_rules/aws_cloudtrail_password_policy_discovery.yml

```diff
@@ -0,0 +1,56 @@
+AnalysisType: rule
+Description: This detection looks for *AccountPasswordPolicy events in AWS CloudTrail logs. If these events occur in a short period of time from the same ARN, it could constitute Password Policy reconnaissance.
+DisplayName: AWS CloudTrail Password Policy Discovery
+Enabled: false
+Filename: aws_cloudtrail_password_policy_discovery.py
+Reports:
+    MITRE ATT&CK:
+        - TA0007:T1201
+Severity: Info
+Tests:
```

Comment:
Are these tests representative of actual log structure?

Reply:
Looks correct according to this, for fields referenced by the rule: https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-log-file-examples.html#cloudtrail-log-file-examples-iam